openedx_external_enrollments/external_enrollments/icc_external_enrollment.py

```diff
@@ -189,12 +192,8 @@ def _get_icc_user_from_xml_response(self, response, log_details, method_type):
         except (TypeError, KeyError):
             LOG.warning('User was not found. Proceeding to create it.')
         except Exception as error:  # pylint: disable=broad-except
-            exception_message = response['EXCEPTION']['MESSAGE'] if response.get('EXCEPTION') else error
-            log_details['response'] = {
-                'error': ('Failed to retrieve ICC user from response. Reason: %s' % exception_message),
-            }
-
-            LOG.error('Failed to retrieve ICC user from response. Reason: %s', exception_message)
+            LOG.error('Failed to retrieve ICC user from response. Reason: %s',
+                      response['EXCEPTION']['MESSAGE'] if response.get('EXCEPTION') else error)
```

Comment:
Move the closing parenthesis to a new line. @ivanvgh

Reply:
Moved.